content/docs/user_guide.md

```diff
@@ -0,0 +1,741 @@
++++
+title = "User Guide"
+description = "In-depth instructions for using Kubeflow"
+weight = 10
+toc = true
+bref= "Detailed description of Kubeflow usage"
++++
+
+# Using Kubeflow
```

Comment:
One last request. Looking at the [rendered page](https://deploy-preview-60--competent-brattain-de2d6d.netlify.com/docs/user_guide/) I think we should remove line 9 which looks like a redundant header.

Reply:
@abhi-g  Please have a look now